src/Components/Facility/Investigations/ShowInvestigation.tsx

```diff
@@ -143,25 +147,77 @@ export default function ShowInvestigation(props: any) {
   }
 
   return (
-    <div className="mx-auto max-w-7xl px-4">
-      <PageTitle
-        title={t("investigations")}
-        className="mx-3 md:mx-4"
-        crumbsReplacements={{
-          [facilityId]: { name: patientData?.facility_object?.name },
-          [patientId]: { name: patientData?.name },
-        }}
-        backUrl={`/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}`}
-      />
-      <InvestigationTable
-        title={`ID: ${sessionId}`}
-        data={state.initialValues}
-        isDischargedPatient={!!consultation?.discharge_date}
-        changedFields={state.changedFields}
-        handleValueChange={handleValueChange}
-        handleUpdateCancel={handleUpdateCancel}
-        handleSave={handleSubmit}
-      />
-    </div>
+          <PrintPreview title={`Investigation Report for ${patientData?.name}`}
+          >
+        <div id="section-to-print">
+          
+          <div
+          className="print:w-[1100px]">
```

Comment:
Prefer rem values over hardcoded arbitrary pixel values. This will result in different results in different browser/system zoom levels.